sunpy/util/decorators.py

```diff
@@ -44,6 +44,10 @@ def new_function():
         place of the deprecated object. The deprecation warning will
         tell the user about this alternative if provided.
     """
+    major = int(since.split('.')[0]) + 1
+    minor = 1
```

Comment:
```python
since_major, since_minor = since.split('.')[:2]
since_lts = since_minor == 0
if since_lts:
    major = since_major
    minor = since_minor + 1
else:
    major = since_major + 1
    minor = 1
```